File: framework/src/sbt-plugin/src/sbt-test/routes-compiler-plugin/routes-compilation/app/controllers/module/ModuleController.scala
/*
 * Copyright (C) 2009-2018 Lightbend Inc. <https://www.lightbend.com>
 */
package controllers.module

import play.api.mvc._
import javax.inject.Inject

class ModuleController @Inject()(c: ControllerComponents) extends AbstractController(c) {
  def index = Action {
    Ok
  }
}

File: dev-mode/sbt-plugin/src/sbt-test/play-sbt-plugin/routes-compiler-routes-compilation-java/app/controllers/module/ModuleController.java
/*
 * Copyright (C) from 2022 The Play Framework Contributors <https://github.com/playframework>, 2011-2021 Lightbend Inc. <https://www.lightbend.com>
 */

package controllers.module;

import jakarta.inject.Inject;
import play.mvc.Controller;
import play.mvc.Result;

public class ModuleController extends Controller {

  @Inject
  public ModuleController() {}

  public Result index() {
    return ok();
  }
}
